apps/artifactory/artifactory-operator/config/default/manager_auth_proxy_patch.yaml

```diff
@@ -0,0 +1,26 @@
+# This patch inject a sidecar container which is a HTTP proxy for the
```

Comment:
are these configs static and for this version specific? is there need to parametrize them?

Reply:
this is actually part of the scaffolding of the new operator version, so i didn't write this specific bit